code/modules/surgery/organs/lungs.dm

```diff
@@ -298,15 +298,9 @@
 	// Nitryl
 		var/nitryl_pp = breath.get_breath_partial_pressure(breath_gases[/datum/gas/nitryl][MOLES])
 		if (prob(nitryl_pp))
-			to_chat(H, "<span class='alert'>Your mouth feels like it's burning!</span>")
-		if (nitryl_pp >40)
-			H.emote("gasp")
-			H.adjustFireLoss(10)
-			if (prob(nitryl_pp/2))
-				to_chat(H, "<span class='alert'>Your throat closes up!</span>")
-				H.silent = max(H.silent, 3)
-		else
-			H.adjustFireLoss(nitryl_pp/4)
+			H.emote("burp")
+			H.adjustOrganLoss(ORGAN_SLOT_LUNGS, 3)
+			to_chat(H, "<span class='notice'>You feel a burning sensation in your throat</span>")
```

Comment:
This should scale with the pressure, also do you really want them burping here?

Reply:
Burping is now rolled independent of lung damage. I've changed the lung damage to scale until your breathing over 40% nitryl, where it caps at 25 per breath as not to instantly destroy lungs breathing pure nitryl out of a tank.